src/validators.js

```diff
@@ -95,3 +95,9 @@ export function validateAlpha(attribute, value) {
 export function validateAlphaNum(attribute, value) {
   return (typeof value === 'string' || typeof value === 'number') && is.alphaNumeric(value)
 }
+
+export function validateRegex(attribute, value, parameters) {
```

Comment:
You can follow logic implemented here https://github.com/laravel/framework/blob/5.4/src/Illuminate/Validation/Concerns/ValidatesAttributes.php#L1042